xml/art_admin_slemicro_cockpit.xml

```diff
@@ -125,945 +140,2095 @@
+     <imagedata  fileref="cockpit_networking_plugin.png" width="100%"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="cockpit_networking_plugin.png" width="100%"/>
+    </imageobject>
+    <textobject role="description"><phrase>Cockpit networking</phrase>
+    </textobject>
+   </mediaobject>
+  </figure>
+  <section xml:id="sec-networking-firewall">
+   <title>Firewall</title>
+   <important>
+    <title>Firewall and Podman</title>
+    <para>
+     Using firewall along with Podman may result in missing Podman-relateds
+     firewall rules after reloading the firewalld service. Therefore, it is
+     recommended to keep the firewall in its default setting (disabled) if you
+     intend to use Podman.
+    </para>
+   </important>
+   <para>
+    The <guimenu>Firewall</guimenu> view enables you turn on the firewall,
+    which is off by default. After turning it on, you can edit firewall rules
+    and zones by clicking <guimenu>Edit rules and zones</guimenu>
+   </para>
+   <figure xml:id="fig-firewall">
+    <title>Firewall settings</title>
+    <mediaobject>
+     <imageobject role="fo">
+      <imagedata  fileref="cockpit_firewall_settings.png" width="100%"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="cockpit_firewall_settings.png" width="100%"/>
+     </imageobject>
+     <textobject role="description"><phrase>Cockpit firewall</phrase>
+     </textobject>
+    </mediaobject>
+   </figure>
+   <para>
+    If you need to define a zone on top of the currently available, click
+    <guimenu>Add zone</guimenu>. In the <guimenu>Add zone</guimenu> window
+    select the <guimenu>Trust level</guimenu>. Each trust level of network
+    connections has a predefined set of included services (the Cockpit serivce
+    is included in all trust levels). You can also allow addresses from the
+    entire subnet within the particular trust level, or you can define a
+    specific range of allowed IP addresses - as a comma separated list.
+   </para>
+   <figure xml:id="fig-firewall-zones">
+    <title>Add firewall zone</title>
+    <mediaobject>
+     <imageobject role="fo">
+      <imagedata  fileref="cockpit_firewall_zones.png" width="100%"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="cockpit_firewall_zones.png" width="100%"/>
+     </imageobject>
+     <textobject role="description"><phrase>Cockpit firewall</phrase>
+     </textobject>
+    </mediaobject>
+   </figure>
+  </section>
+  <section xml:id="sec-networking-interfaces">
+   <title>Interfaces</title>
+   <para>
+    From the <guimenu>Interfaces</guimenu> part, you can add, modify or delete
+    VLANs, bridges or bond interfaces.
+   </para>
+   <section xml:id="sec-networking-bridges">
+    <title>Managing bonds</title>
+    <para>
+     A bond interface is a combination of several network interfaces into one
+     bond. Network bonding can increase performance by increasing the network
```

Comment:
Depending on the _Mode_, network bonding can (...)

(...) It can also increase fault tolerance by keeping overall connectivity even when some of the interfaces stop working.